plugins/item_tasks/web_client/templates/selectSingleFileTaskWidget.pug

```diff
@@ -0,0 +1,18 @@
+.modal-dialog
+  .modal-content
+    form#g-select-task-form.modal-form(role="form")
+      .modal-header
+        button.close(data-dismiss="modal", aria-hidden="true", type="button") &times;
+        h4.modal-title Select Task
+      .modal-body
+        .g-task-container
+          .g-task-list-container
+          hr
+          h4 Selected task
+          .g-task-selected-container
+            | No task selected
+        .g-validation-failed-message
+      .modal-footer
+        a.btn.btn-small.btn-default(data-dismiss="modal") Close
+        button.g-submit-select-task.btn.btn-small.btn-primary(type="submit")
```

Comment:
I think some of these class names could be cleaned up to follow a more hierarchical scheme. However, since this is a brand new file, we're between a release cycle, and I know you're planning on making changes to this file soon in a subsequent PR (to remove the `g-task-selected-container` entirely), I think we can change it in the next PR without compatibility worries.